src/pages/WishList.jsx

```diff
@@ -0,0 +1,34 @@
+import WishListHeader from 'components/wishlist/WishListHeader';
+import { client } from 'libs/api';
+import styled from 'styled-components';
+import React, { useState, useEffect } from 'react';
+import WishListInfo from 'components/wishlist/WishListInfo';
+
+function WishList() {
+  const [categoryList, setCategoryList] = useState([]);
+
+  const getCategoryList = async () => {
+    const { data } = await client.get('/category');
+    setCategoryList(data);
+    console.log(data);
+  };
+
+  useEffect(() => {
+    getCategoryList();
+  }, []);
+
+  return (
+    <StyledCategory>
+      <WishListHeader></WishListHeader>
+      <WishListInfo list={categoryList}></WishListInfo>
```

Comment:
```suggestion
      <WishListHeader />
      <WishListInfo list={categoryList} />
```

여기도 self-closing 하면 될듯해!!